README.md

```diff
@@ -1,219 +1,113 @@
-            </tr>
-            <tr>
-                <td>DELETE</td>
-                <td>/location/:id</td>
-                <td>Deleta o local</td>
-                <td>Apenas administradores</td>
-            </tr>
-        </tbody>
-        </table>
-        <h1>Rotas de Categorias</h1>
-        <table>
-        <thead>
-            <tr>
-                <th>Método</th>
-                <th>Endpoint</th>
-                <th>Responsabilidade</th>
-                <th>Autenticação</th>
-            </tr>
-        </thead>
-        <tbody>
-            <tr>
-                <td>POST</td>
-                <td>/category</td>
-                <td>Cria uma categoria</td>
-                <td>Apenas administradores</td>
-            </tr>
-            <tr>
-                <td>GET</td>
-                <td>/categories</td>
-                <td>Lista todas as categorias</td>
-                <td>Não necessita token</td>
-            </tr>
-            <tr>
-                <td>GET</td>
-                <td>/category/:id</td>
-                <td>Lista uma categoria por id</td>
-                <td>Não necessita token</td>
-            </tr>
-            <tr>
-                <td>PUT</td>
-                <td>/category/:id</td>
-                <td>Atualiza uma categoria</td>
-                <td>Apenas administradores</td>
-            </tr>
-            <tr>
-                <td>DELETE</td>
-                <td>/category/:id</td>
-                <td>Deleta uma categoria</td>
-                <td>Apenas administradores</td>
-            </tr>
-        </tbody>
-    </table>
-</body>
-</html>
+
+#   Gerenciamento Eventos Culturais
+## Introdução
+
+Esta aplicação consiste em um cadastro de eventos Culturais, onde é visado a criação de uma plataforma robusta para o gerenciamento de eventos. A plataforma oferecerá aos organizadores de eventos a capacidade de criar e listar seus eventos de maneira eficiente. Além disso, proporcionará aos participantes a oportunidade de explorar, pesquisar e filtrar eventos com base em critérios específicos como categorias, locais e datas.
+
+# Funcionalidades
+- Criar eventos, autenticação 
+- Listar seus eventos
+- Filtrar eventos
+
+# Tecnologias utilizadas
+- NodeJS  
+- Express                     
+- Prisma
+- PostgreSQL
+- Insomnia
+- API REST
+
+# Diagrama de Relacionamento
+<img align="center" height="550" width="780" src="DER.png" alt="Imagem do diagrama de Relacionamento">
+
+# Como executar
+Antes de tudo, clone o repositório em sua máquina usando: 
+```bash
+  git clone https://github.com/AvantiHeadHunters/gerenciamento-eventos-culturais.git
+```
+Depois instale as dependências rodando o seguinte comando:
+
+```bash
+npm install
+```
+Aqui será preciso criar um arquivo `.env` seguindo o exemplo do arquivo `.env.example`.
+<br>
+Preencha o arquivo para configurar o banco de dados local. 
+
+```bash
+DATABASE_URL=postgresql://user:password@localhost:5432/nome_do_banco
```

Comment:
não acho necessário dizer quais são as variáveis de ambientes a serem utilizadas, tendo em vista que em um projeto maior, podem ter inúmeras variáveis... mas caso queira realmente deixar de forma explícita, é bom atualizar com a outra variável. se não me engano, é SECRET_JWT, ou algo do tipo :D